go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -221,7 +132,7 @@ func runOvnKube(ctx *cli.Context) error {
 	}
 
 	// creates the clientset
-	clientset, err := kubernetes.NewForConfig(config)
+	clientset, err := kubernetes.NewForConfig(kconfig)
```

Comment:
It seems to be the same as before, it should be possible to use NewClientset here too.

Reply:
@alinbalutoiu I actually do this in a subsequent commit "go-controller: consolidate common kubeclient creation code". If that's not what you're thinking of, let me know.

Reply:
@dcbw yep it's fine, you sent the new commits while I was reviewing the patches. This review got marked automatically as outdated.